File: ago-dic-2018/José Horacio Góngora Treviño/Parcial_1/Pelicula.java

public class Pelicula
{
	private String nombre;
	private int hrs;
	private int min;
	
	public Pelicula(String nombre, int hrs, int min)
	{
		super();
		this.nombre=nombre;
		this.hrs=hrs;
		this.min=min;
	}
	
	public String getNombre()
	{
		return nombre;
	}
	
	public void setNombre(String nombre)
	{
		this.nombre=nombre;
	}
	
	public int getHrs()
	{
		return hrs;
	}
	
	public void setHrs(int hrs)
	{
		this.hrs=hrs;
	}
	
	public int getMin()
	{
		return min;
	}
	
	public void setMin(int min)
	{
		this.min=min;
	}
	
	public String toString()
	{
		return nombre + "," + hrs+":"+min;
	}
}

File: ago-dic-2020/Luis Enrique Cázares Martínez/1er Parcial/Pelicula.py
class pelicula():
    def __init__(self, nombre, hrs, min):
        self.nombre = nombre
        self.hrs = hrs
        self.min = min

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getHrs(self):
        return self.hrs

    def setHrs(self, hrs):
        self.hrs = hrs

    def getMin(self):
        return self.min

    def setMin(self, min):
        self.min = min

    def sumaHrsMin(self,hrs, min):
        suma = hrs + min
        return suma

    def __str__(self):
        return f"Pelicula -> {self.nombre} Duración -> {self.hrs}:{self.min}"